src/_includes/assets/css/screen.css

```
@@ -1945,84 +1945,107 @@ pre.runkit-element {
 /* ---------------------------------------------------------- */
 
 .site-footer {
-  position: relative;
   color: var(--gray85);
   background: var(--gray05);
   line-height: 1.6;
+  padding: 40px 15px;
+  font-size: 16px;
 }
 
-.footer-container {
-  margin-right: auto;
-  margin-left: auto;
-  padding-left: 15px;
-  padding-right: 15px;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  font-size: 16px;
-  overflow-x: hidden;
+.footer-top {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(30.5em, 1fr));
+}
+
+@media (max-width: 500px) {
+  .footer-top {
+    grid-template-columns: repeat(auto-fit, minmax(19em, 1fr));
+  }
 }
 
-.footer-container p {
+.footer-top,
+.footer-bottom {
+  width: min(100%, 1300px);
+  margin-inline: auto;
+}
+
+.site-footer p {
   margin: 0 0 1.45rem;
   line-height: 30px;
+  font-size: 16px;
 }
 
-.footer-container a {
+.site-footer a {
   color: var(--gray85);
   text-decoration: none;
+  padding: 2px;
 }
 
-.footer-container .col-header {
-  flex: 0 0 100%;
-  padding-bottom: 15px;
+.site-footer a:hover {
+  text-decoration: underline;
+}
+
+.site-footer .col-header,
+.trending-guides .col-header {
+  text-align: center;
   font-weight: 700;
   font-size: 16px;
-  text-align: center;
   padding: 0 15px 15px;
+  margin-bottom: 0;
+}
+
+.trending-guides-articles {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(11em, 1fr));
+  grid-column-gap: 3rem;
+  grid-row-gap: 0.5rem;
+}
+
+.trending-guides-articles li {
+  padding: 0;
+  margin: 0;
+}
+
+@media (max-width: 1400px) {
+  .trending-guides-articles {
+    grid-template-columns: repeat(auto-fit, minmax(12em, 1fr));
+  }
+}
+
+@media (max-width: 800px) {
+  .trending-guides-articles {
+    grid-template-columns: repeat(auto-fit, minmax(13em, 1fr));
+  }
 }
 
 .footer-row {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
   margin: 0;
 }
 
 .footer-col {
-  display: flex;
-  flex-direction: column;
-  flex: 0 0 100%;
-  padding-left: 15px;
-  padding-right: 15px;
+  padding-inline: 15px;
   font-size: 16px;
 }
 
-.footer-right,
-.footer-left {
-  display: flex;
-  flex-direction: column;
-  flex: 0 0 100%;
-}
-
 .footer-col a {
   padding: 5px 0;
 }
 
 .footer-desc-col {
-  flex: 1 0 90%;
-  display: flex;
-  flex-direction: column;
-  padding-left: 15px;
-  padding-right: 15px;
+  padding-inline: 15px;
   margin-bottom: 30px;
+  align-self: center;
```

```suggestion
    align-self: start;
    margin-top: 24px;
```

centering the description column is a bug on learn.
This change should send the footer to the top and align it with the trending articles.